src/main/menu.js

```diff
@@ -92,13 +97,24 @@ fluid.defaults("gpii.app.menuInAppDev", {
             singleTransform: {
                 type: "fluid.transforms.free",
                 func: "gpii.app.menu.generateMenuTemplate",
-                args: ["{that}.model.showPSP", "{that}.model.keyedInSnapset", "{that}.options.locales", "{that}.options.snapsets", "{that}.model.preferenceSetsMenuItems", "{that}.model.keyOut", "{that}.options.exit"]
+                args: [
+                    "{that}.model.showQSS",
```

Comment:
This has become really huge and would benefit a lot from being converted into a structured "options" object

Reply:
Can you please clarify what you mean?

Reply:
I mean sending a structure organised like 
````
{
    showPSP: "{that}.model.showPSP",
    keyedInSnapset: "{that}.model.keyedInSnapset"
````
etc

Reply:
Sorry, looking at the implementation now I see its explicit purpose is to process an ordered list, please ignore this comment